_posts/2019-01-20-artsy-engineering-hiring.md

```diff
@@ -0,0 +1,300 @@
+completed their write-ups, the interviewers debrief and reflect on how to do a better job next time. Their feedback
+is used by the hiring manager to decide whether to move onto reference checks.
+
+Quoting again from our former Director of Web's blog post:
+
+> Artsy believes that 'references are not a defense against hiring poorly, they are a way to hire great people'.
+
+Artsy's reference checks are in-depth and deserve their own blog post; they are _key_ to our hiring process. We
+know that job interviewers only evaluate how good someone is at interviewing, so we put a larger emphasis than most
+companies on references. The most accurate predictor of future job performance is past job performance, not how
+well someone can perform in an interview.
+
+If we decide to hire the candidate, we make them a job offer. Artsy offers what we think is a fair wage based on
+the local market and the candidate; we do not low-ball candidates and we don't negotiate on compensation.
+
+## What's wrong with typical hiring practices?
+
+There are many tactics for assessing a candidate's technical abilities, but we've found that many are unfair to the
+candidate. Some strategies put unnecessary pressure on the candidate. Some select against qualified candidates who
+have competing responsibilities outside work. Some unwittingly weed out underrepresented applicants, even at a time
+when companies are trying to diversify their teams.
+
+### In-person coding challenges
+
+The intention of in-person coding challenges is to verify that the engineer can "actually write code." This
+strategy puts excessive pressure on the candidate to perform in front of an audience. This is usually not a good
+reflection of what the candidate would be doing if they were hired. Sometimes it **is** a reflection of the
+stressful conditions on the team, and the act of applying pressure to the candidate is intentional, to measure
+their ability to handle it. In either case, we don't feel like this is how we want to measure engineers; it just
+don't reflect reality.
+
+### Whiteboard interviews
+
+One intention of whiteboard interviews is to reduce the stress on the candidate, because they don't have to worry
+about code syntax while under a microscope. These types of interviews still lead to stressful conditions, though,
+and they don't provide a good measure of what makes a great teammate or even a great developer. Again, sometimes
+the pressure is intentional, to see how the candidate reacts.
+
+It can be very difficult to find a problem that is succinct enough for a whiteboard exercise but still reflective
+of the work the candidate will actually be doing on the job. The ability to write an algorithm to search a binary
+tree might be reflective of whether a candidate has a traditional Computer Science degree, but doesn't say much
+about their ability to build complex interfaces or streamline performance. Questions like this can eliminate
```

Comment:
> The ability to write an algorithm to search a binary tree
> but doesn't say much about their ability to [...] streamline performance

Not to be too pedantic but this is probably a poor example because binary trees are very much related to streamlining performance and are often used when constructing complex interfaces. Data structures have applications all over the map -- but they're just one tiny portion of the whole of being a software engineer and working effectively on a team. 

> but doesn't say much about ... 

It might be good to expand this outside of technical terms and bring in something about effective communication or teamwork or perhaps even learning.